fhir-validation/src/main/java/com/ibm/fhir/validation/FHIRValidator.java

```diff
@@ -336,7 +336,7 @@ private void validate(FHIRPathElementNode elementNode) {
                 if (isAbsolute(url)) {
                     if (FHIRRegistry.getInstance().hasResource(url, StructureDefinition.class)) {
                         constraints = new ArrayList<>(constraints);
-                        constraints.add(createConstraint("generated-ext-1", Constraint.LEVEL_RULE, Constraint.LOCATION_BASE, "Extension must conform to definition '" + url + "'", "conformsTo('" + url + "')", false, true));
+                        constraints.add(createConstraint("generated-ext-1", Constraint.LEVEL_RULE, Constraint.LOCATION_BASE, "Extension must conform to definition '" + url + "'", "conformsTo('" + url + "')", "", false, true));
```

Comment:
What does the empty string signify ?

Reply:
answered my own question

Reply:
That's actually something I was just looking at. Whether I should use UNKNOWN or make up a URL that means "this was created in the validator itself". Any thoughts?

Reply:
Added a public constant `SOURCE_VALIDATOR` with a value of `http://ibm.com/fhir/validation/FHIRValidator`.